### PR TITLE
Fix Assembly files

### PR DESCRIPTION
**Pull request by MrBrain295**
## Adding a language

- [x] The code displays "Hello World"
- [ ] I have updated the readme to include the new language
- [ ] I have incremented the language count in the readme

### CODE REVIEW

**Comment by MrBrain295** [comment]
@Richienb wasn't there a commit making every this say "Hello World"?

File: a/Assembler Bootsect.asm [hunk 1]
```diff
@@ -19,4 +19,4 @@ print_string:
 	ret
 	
 	times	510-($-$$)	db	0
-	dw	0xAA55
\ No newline at end of file
+	dw	0xAA55
```
Comment (MrBrain295) [inline a/Assembler Bootsect.asm]: IDK how this happened

**Comment by MrBrain295** [comment]
Part of #1000

File: a/Assembler 8048 videopac.asm [hunk 2]
```diff
@@ -44,7 +44,7 @@ nextchar:
 loop:
 	jmp 	loop                            ; just wait and do nothing
 
-; 'HELLO WORLD' (ascii not supported by assembler)
+; 'Hello World' (ascii not supported by assembler)
```
Comment (Richienb) [inline a/Assembler 8048 videopac.asm:47]: This is actually just a comment - the string that is printed also needs to be encoded.

**Review by Richienb — CHANGES_REQUESTED** [review]

File: a/Assembler 8048 videopac.asm (hunk 2 above)
Comment (MrBrain295) [reply a/Assembler 8048 videopac.asm:47]: Sorry, I don’t know how to do that.

Comment (MrBrain295) [reply a/Assembler 8048 videopac.asm:47]: @Richienb do you?

Comment (Richienb) [reply a/Assembler 8048 videopac.asm:47]: Since ASCII is not supported by assembler, there is no way to do it so this comment should just be left as is.

Comment (MrBrain295) [reply a/Assembler 8048 videopac.asm:47]: Ok

**Comment by MrBrain295** [comment]
@Richienb done ✅

**Comment by MrBrain295** [comment]
@Richienb does anything else need fixed?

File: a/Assembler 8048 videopac.asm (hunk 2 above)
Comment (MrBrain295) [reply a/Assembler 8048 videopac.asm]: done

**Review by Richienb — APPROVED** [review]

**Comment by MrBrain295** [comment]
Thanks

**Pull request merged**